objects/date.js

```diff
@@ -223,45 +246,80 @@ module.exports = class Date extends global.Date {
 		return this;
 	}
 
+	/**
+	 * Clones the current date. The new object is independent of the old one.
+	 * @returns {sb.Date}
+	 */
 	clone () {
 		return new this.constructor(this);
 	}
 
+	/**
+	 * @param {number} y Number of years to add.
+	 * @returns {sb.Date}
+	 */
 	addYears (y) {
 		this.year += y;
 		return this;
 	}
 
+	/**
+	 * @param {number} m Number of months to add.
+	 * @returns {sb.Date}
+	 */
 	addMonths (m) {
 		this.month += m;
 		return this;
 	}
 
+	/**
+	 * @param {number} d Number of days to add.
+	 * @returns {sb.Date}
+	 */
 	addDays (d) {
 		this.day += d;
 		return this;
 	}
 
+	/**
+	 * @param {number} h Number of hours to add.
+	 * @returns {sb.Date}
+	 */
 	addHours (h) {
 		this.hours += h;
 		return this;
 	}
 
+	/**
+	 * @param {number} m Number of minutes to add.
+	 * @returns {sb.Date}
+	 */
 	addMinutes (m) {
 		this.minutes += m;
 		return this;
 	}
 
+	/**
+	 * @param {number} s Number of seconds to add.
+	 * @returns {sb.Date}
+	 */
 	addSeconds (s) {
 		this.seconds += s;
 		return this;
 	}
 
+	/**
+	 * @param {number} ms Number of milliseconds to add.
+	 * @returns {sb.Date}
+	 */
 	addMilliseconds (ms) {
 		this.milliseconds += ms;
 		return this;
 	}
 
+	/**
+	 * @returns {string} The day of the week as text. For example "Monday"
```

Comment:
should mention this is the English day of the week and similar to the suffix earlier, should return an enum or its own `@typedef`.